### PR TITLE
(python) don't include `(` `)` in functions `params` match

### PR DESCRIPTION
**Pull request by mondeja**
Python functions parameters before this pull are including `(` and `)` characters. This not is the desired behaviour as commented in pull #2489.

### CODE REVIEW

**Comment by yyyc514** [comment]
Please add changelog entry.

File: test/markup/python-repl/sample.expect.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ foo = 42"
 <span class="hljs-meta">&gt;&gt;&gt;</span> <span class="python"><span class="hljs-string">"""</span></span>
 <span class="hljs-meta">...</span> <span class="python"><span class="hljs-string">abc</span></span>
 <span class="hljs-meta">...</span> <span class="python"><span class="hljs-string">"""</span></span>
-<span class="hljs-meta">&gt;&gt;&gt;</span> <span class="python"><span class="hljs-function"><span class="hljs-keyword">def</span> <span class="hljs-title">test</span><span class="hljs-params">()</span>:</span></span>
+<span class="hljs-meta">&gt;&gt;&gt;</span> <span class="python"><span class="hljs-function"><span class="hljs-keyword">def</span> <span class="hljs-title">test</span>(<span class="hljs-params"></span>):</span></span>
```
Comment (yyyc514) [inline test/markup/python-repl/sample.expect.txt:14]: I'll accept as is, but would it be too much trouble to handle the edge case of () so it doesn't produce noise?

It'd prolly be something like:

```
  var PARAMS = {
    className: 'params',
    variants: [
     {begin: /\(/, end: /\)/, excludeBegin: true, excludeEnd: true, contains: ['self', PROMPT, NUMBER, STRING, hljs.HASH_COMMENT_MODE]},
     {begin: /\(\s*\)/, skip: true, className: null }
    ]
    
  };
```

Or some such.

File: CHANGES.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ Parser Engine Changes:
 
 Language Improvements:
 
+- enh(python) Exclude parens from functions params (#2490) [Álvaro Mondéjar][]
```
Comment (yyyc514) [inline CHANGES.md:25]: You also have to add your url in the footnotes around line 60 or so.

File: test/markup/python-repl/sample.expect.txt (hunk 1 above)
Comment (mondeja) [reply test/markup/python-repl/sample.expect.txt]: Ok, it's included.

**Comment by yyyc514** [comment]
@mondeja Thanks for your contribution!
